interface/forms/dm2_fu_ap_01/info.txt

```diff
@@ -0,0 +1 @@
+DM2_FU_AP_01
```

Comment:
would make the display name more human readable

![Screenshot from 2021-11-23 06-52-45](https://user-images.githubusercontent.com/5618372/143019551-d842b2e5-b468-49de-ab62-ac0579b6e99b.png)

Reply:
Agree with @stephenwaite that the form names should be human readable (ie. what is in the info.txt files). When that is addressed, then this PR can be brought into the codebase.